### PR TITLE
Only use videoplayer.limitguiupdate on Pi and IMX

### PR DESCRIPTION
**Pull request by fritsch**
Fix up of: https://github.com/xbmc/xbmc/pull/6898

### CODE REVIEW

File: xbmc/Application.cpp [hunk 1]
```diff
@@ -2482,8 +2482,14 @@ void CApplication::FrameMove(bool processEvents, bool processGUI)
   {
     m_skipGuiRender = false;
     int fps = 0;
+
+    #if defined(TARGET_RASPBERRY_PI) || defined(HAS_IMXVPU)
```
Comment (FernetMenta) [inline xbmc/Application.cpp]: please, please, please move those to the very left. what is the benefit of mixing code with pre-processor statements?

Comment (fritsch) [inline xbmc/Application.cpp]: coding in a 80 cols terminal - you have to scroll on the very left to see what is going on ;-)

File: xbmc/Application.cpp [hunk 2]
```diff
@@ -2482,8 +2482,14 @@ void CApplication::FrameMove(bool processEvents, bool processGUI)
...
+    // This code reduces rendering fps of menus when playing videos in fullscreen mode
```
Comment (FernetMenta) [inline xbmc/Application.cpp]: actually it reduces fps for entire gui not only menus

**Comment by fritsch** [comment]
Done as you wish master :-)

**Comment by FernetMenta** [comment]
thanks, looks good to me.

**Pull request merged**